CHANGELOG.md

```diff
@@ -1,12 +1,10 @@
 # Change Log
 
-## Unreleased
-
-### Changes
```

Comment:
Per http://keepachangelog.com/ (referenced in our other OSS projects' change logs), we should keep the subsections (probably "Added" and "Fixed" here).